src/metadata/trackmetadata.h

```diff
@@ -114,6 +122,13 @@ class TrackMetadata {
         m_bitrate = bitrate;
     }
 
+    inline int getRating() const {
+        return m_rating;
+    }
+    inline void setRating(int rating) {
+        m_rating = rating;
```

Comment:
I will remove the rating from the metadata. We can re-add it later after the mapping of the internal Mixxx rating to the various tag ratings is defined.